src/Voron/StorageEnvironmentOptions.cs

```diff
@@ -1233,7 +1233,8 @@ public int TimeToSyncAfterFlushInSec
         public bool SkipChecksumValidationOnDatabaseLoading { get; set; }
 
         public int MaxNumberOfRecyclableJournals { get; set; } = 32;
-
+        public bool DiscardVirtualMemory { get; set; }
```

Comment:
Maybe set default to true? Some tests are creating StorageEnv directly. Should not matter much, but I think its better to have it turn on.

Reply:
done